### ISSUE TITLE
Codewind no longer auto-detect appsody project

### ISSUE THREAD

**Issue by makandre** [eclipse/codewind#445]
**Codewind version:** 0.4.0
**OS:**

**Che version:**
**IDE extension version:**
**IDE version:**
**Kubernetes cluster:**

**Description:**

I believe when we switched over to using installer CLI for project validation and creation, we are no longer able to auto-detect an appsody project, as evident by the following test:

```
andrews-mbp:config andrewmak$ /var/folders/c8/12yjw3yn40gcmw3mpzv0sj6c0000gn/T/codewind-installer project /opt/Repos/Tempest/codewind/codewind-workspace/knjavamp
...
{"status":"success","projectPath":"/opt/Repos/Tempest/codewind/codewind-workspace/knjavamp","result":{"language":"java","projectType":"docker"}}
```

Note the `"projectType": "docker"` in the response.  As a result, when we import an existing appsody project into IDEs, user will *always* get prompted to pick the project type.

**Steps to reproduce:**

1. create an appsody project
2. remove appsody project from Codewind, but leave project dir behind
3. Import the project back into Codwind
4. Get prompted for the project type

**Workaround:**

**Comment by ebuckle** [comment]
I'm looking into this

**Comment by makandre** [comment]
Hey @ebuckle, I actually have some thing in the works for #442, which is also going to fix this problem.

**Comment by ebuckle** [comment]
Sure. It would be possible to fix this through the Appsody logic in the CLI but your solution is much cleaner for the long term.

**Comment by jagraj** [comment]
Looks like appsody project imports are failing to set language type intermittently even though I selected language type "Java". 

Steps to recreate.

1. Create appsody Microprofile Java project.
2. Wait until it builds and running.
3. Remove project from Eclipse "codewind explorer view".
4. Import again, with "Appsody" and "Java" options and the language not being set.

This happens 50% of the time and this is intermittent. We see mixed results.  You can see the screenshot below, the image for Java project is not for Java and language is set "Unknown" in overview page.

This seems to be regression and we need to deliver the fix for this as we have conferences scheduled in the coming weeks.

![Screen Shot 2019-09-18 at 5 16 22 PM](https://user-images.githubusercontent.com/713849/65186667-152ff900-da38-11e9-8883-a886dfd53556.png)

**Comment by makandre** [comment]
The language select problem is likely not directly caused by this issue (not auto-detecting appsody project) but it is exposed by it (since now you always have to choose the type and language when importing an appsody project)

The language select problem is possibly fixed by https://github.com/eclipse/codewind-eclipse/pull/309; @eharris369 can confirm.

**Comment by eharris369** [comment]
The language issue will be fixed by eclipse/codewind-eclipse#309.  Just waiting for John to review.
